### ISSUE TITLE
Only generate em-based media-queries even when breakpoint is set in pixel

### ISSUE THREAD

**Issue by Nicolas-Bouteille** [at-import/breakpoint#100]
Hi,

First of all, since this is my first issue posting on any Git Hub project, I apologize if I am not doing things 100% as I should. I've searched for similar suggestions but did not find any.

So this is more a feature idea... that came to me after reading a lot about em-based breakpoints and good practices. But I might be missing something here because I'm sure this would have been already done if possible...

So...

I definitely agree that all media-queries should be em-based not pixel-based for better experience when the user zooms-in or increases the default browser font-size.

I definitely agree that we should stop having device-based breakpoints but content-based breakpoints instead so that the responsive experience works not only today but also tomorrow when devices' size will evolve.

However, I do not agree with the fact that we should stop thinking in pixels. As far as I'm concerned, in order to test the responsive experience, I use something like Firefox adaptive view, start small, increase until it looks like shit... add a breakpoint. But the value I see is in pixels! So I want to go add my breakpoint in pixels. But I want it to be automatically converted into an em-based media-query for the reasons stated above.

I think the default behavior should now be that @include breakpoint() should render em-based media-queries only, even when the input value is set in pixels.

For people wanting to keep pixel-based media queries, there could be an argument passed, or a special pixel-breakpoint() mixin...

So to sum up, I think we should be able to think and write pixel-valued breakpoints that would generate em-based media-queries by default.

ex: @include breakpoint(480px) should generate @media(min-width: 30em)

What did I miss? 
Am I even right to assume a 480px breakpoint will always be equivalent to a 30em breakpoint because it is based on the browser's default font-size which is 16px for all of them? If not, then does this mean a 30em breakpoint will not behave the same way on browser's which base font-size is not 16px? If yes then how can we even use em-based breakpoints then...
